documentation/docs/advanced/ollama.mdx

```diff
@@ -60,7 +60,7 @@ Restart your Khoj server after first run or update to the settings below to ensu
       ```bash
       ollama pull llama3.1
       ```
-   3. Create a new [OpenAI Processor Conversation Config](http://localhost:42110/server/admin/database/openaiprocessorconversationconfig/add) on your Khoj admin panel
```

Comment:
Looks like we should also update `litellm.md`, `lmstudio.md` and `admin.md` similarly.

Reply:
Ah, good catch. Fixed those docs